p2p-tests/seed-and-isolated-peers-test.ts

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env -S node --require ts-node/register/transpile-only
```

Comment:
```suggestion
#!/usr/bin/env -S node --require ts-node/register/transpile-only  --require tsconfig-paths/register
```

Reply:
We should add this file and all the others in this directory to `tsconfig.json`.

Reply:
9ce365e0d0964af452ed04fb89d82972c7cf5eb1